src/components/Navbar.js

```diff
@@ -42,6 +43,11 @@ class Navbar extends Component {
         const result = this.props.userinfo
         const { user } = this.props
         const { isLoggedIn } = this.state
+       
+
+//    let loginButton = localStorage.getItem('token') !== null? this.state.isLoggedIn = true: this.state.isLoggedIn = false;
+//    this.setState({loginButton: loginButton});
```

Comment:
should this be here?

Reply:
This is also the same, warning was to use setState() instead of props to mutate it. So commenting that piece of code and writing the new one did workout for me.

Reply:
Ok so you re-implemented the commented code? if so, then you can remove the commented code @sagar-barapatre

Reply:
Yeah sure !! Will do it